ChangeLog

```diff
@@ -1,3 +1,7 @@
+  * Change the default internal namespace separator from the colon to the pipe
+    (issue #76 from Sebastian Pipping, fixed in pull request #78 from Slávek Banko).
+  * This solves compatibility with libexpat >= 2.4.5 after fix the security
```

Comment:
My vote for dropping the second bullet point to make this one single entry.

Reply:
Yes, it sounds like a good note.

Reply:
Thank you!